src/main/groovy/com/bmuschko/gradle/docker/services/DockerClientService.java

```diff
@@ -0,0 +1,108 @@
+package com.bmuschko.gradle.docker.services;
+
+import com.bmuschko.gradle.docker.tasks.DockerClientConfiguration;
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.core.DefaultDockerClientConfig;
+import com.github.dockerjava.core.DockerClientImpl;
+import com.github.dockerjava.httpclient5.ApacheDockerHttpClient;
+import com.google.common.io.Closer;
```

Comment:
Can you remove the explicit dependency on Guava and handle this differently?

Reply:
Removed and replaced with something similar.

Reply:
> Looks like you missed changing the imports. CI is reporting a compilation error.

I was wondering, tests were running locally and not in CI. Fixed